### DISCUSSION TITLE
Overriding VPN Connection/Association Hours

### DISCUSSION THREAD

**Discussion by klongmitre** [infracost/infracost#2677] [Q&A]
I'm trying to override the hours accounted for when pricing VPN endpoint association and connection hours. It seems doing something like this in a usage file does not work.

```
module.test_vpc.aws_ec2_client_vpn_network_association.test_subnet:
    monthly_hours: 8
```

Any thoughts?

**Comment by alikhajeh1** [comment]
I don't see that resource in https://github.com/infracost/infracost/blob/master/infracost-usage-example.yml (which has the list of all usage based params), so the code needs to be updated to add support for the `monthly_hours` override
